content/routing.md

```diff
@@ -32,7 +32,7 @@ meteor add kadira:flow-router
 
 *Snippet: Adding the Flow Router package*
 
-At the time of writing this guide, Flow Router is at version 2.x.
+At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
```

Comment:
Okay. Sounds good to me.

On Thu, Oct 8, 2015 at 10:51 PM Sashko Stubailo <notifications@github.com>
wrote:

> In content/routing.md
> <https://github.com/meteor/guide/pull/17#discussion_r41541792>:
>
> > @@ -32,7 +32,7 @@ meteor add kadira:flow-router
> >
> >  *Snippet: Adding the Flow Router package*
> >
> > -At the time of writing this guide, Flow Router is at version 2.x.
> > +At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
>
> I feel like that issue thread is probably the right place for people to
> have that discussion, no? I don't think it's possible to defend each design
> decision up front. Alternatively, if you put together an article about Flow
> Router's design decisions, I can link to it here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/guide/pull/17/files#r41541792>.
>